data/ctf_platforms/attack_defense.json

```diff
@@ -45,6 +45,13 @@
       "language": "Python",
       "price": "Free"
     },
+    {
+      "name": "Parrot CTFs",
+      "source": "https://github.com/Parrot-CTFs",
+      "description": "Attack-defence CTF framework",
```

Comment:
I don't see any mention to Attack-defence anywhere, it seems to be a jeopardy platform, could you move it to this category and update the description?